### PR TITLE
Admin: Frontend UI update

### PR DESCRIPTION
**Pull request by tmskrtsz**
# Update admin UI/UX

**Changes in this PR**

- [x] Move from gulp to parcel bundler.
- [x] Update bootstrap from version 3 to latest (4.1).
- [x] Remove bower and re-add plugins through npm.
- [x] Fix certain plugins not working with the way parcels bundles.
- [x] Swap datepicker plugin to a new one. Though might have to change it again as I realize now that the new one doesn't include time.
- [x] Change the base font to Open Sans.
- [x] Update select2 plugin to use the latest version and solve an ajax issue.
- [x] Update summernote to the latest version and have it use bs4.
- [x] Update the design and feel of the main menu.
- [x] Update the navigation bar.
- [x] Add resource generator from [here](https://github.com/shuup/shuup/commit/8f5bda3ee36a9cab50df70df1391740ac43135fa).
- [x] Picotable redesign and UX improvements. 
- [x] Update every component to use new bootstap classes and remove old ones.
- [x] Update the design of the edit view. _In progress..._
- [x] Many of the edit pages have weird issues with broken lines and and bad looking forms.
- [ ] Apply a permanent fix to the temporary workarounds that prevented plugins from loading correctly (summernote tooltip, etc.)
- [x] Update the UI of the shuup-toolbar buttons on the edit pages. Currently it gets too cramped if there are a lot in the same line.
- [x] Update the login page design.
- [x] Update the welcome page design as in the mockups.
- [x] Generate translations for new and altered strings.
- [x] Fix the language changer since that has been broken from very early on.
- [x] Fix responsive issues with the updated support navigation component.
- [x] Fix flake8 errors.

### CODE REVIEW

File: shuup/admin/static_src/base/assets/logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...,-87.618l42.258,0l0,211.465l-42.258,0l0,-87.593l-96.582,0l0,87.593l-42.258,0l0,-211.465Z" style="fill:#041e40;fill-rule:nonzero;"/><path d="M830.8,148l42.258,0l0,87.618l96.582,0l0,-87.618l42.258,0l0,211.465l-42.258,0l0,-87.593l-96.582,0l0,87.593l-42.258,0l0,-211.465Z" style="fill:#041e40;fill-rule:nonzero;"/><path d="M1045.09,148l42.258,0l0,123.872c0,42.895 24.158,55.873 54.324,55.873c29.591,0 54.35,-12.978 54.35,-55.873l0,-123.872l42.258,0l0,123.872c0,69.779 -43.786,92.128 -96.608,92.128c-53.423,0 -96.582,-22.349 -96.582,-92.128l0,-123.872Z" style="fill:#041e40;fill-rule:nonzero;"/><path d="M1268.45,148l42.258,0l0,123.872c0,42.895 24.158,55.873 54.324,55.873c29.591,0 54.325,-12.978 54.325,-55.873l0,-123.872l42.258,0l0,123.872c0,69.779 -43.76,92.128 -96.583,92.128c-53.423,0 -96.582,-22.349 -96.582,-92.128l0,-123.872Z" style="fill:#041e40;fill-rule:nonzero;"/><path d="M1494.81,148l122.243,0c38.953,0 70.947,18.14 70.947,69.478c0,55.597 -31.994,72.509 -70.947,72.509l-79.985,0l0,69.478l-42.258,0l0,-211.465Zm122.243,105.733c15.696,0 28.689,-8.444 28.689,-36.255c0,-24.153 -12.993,-33.223 -28.689,-33.223l-79.985,0l0,69.478l79.985,0Z" style="fill:#041e40;fill-rule:nonzero;"/></g></g></svg>
```
[the remark this answers is not in the document]
Comment (tmskrtsz) [reply shuup/admin/static_src/base/assets/logo.svg]: Actually, shouldn't this be committed?

File: shuup/admin/static_src/base/js/editor.js [hunk 2]
```diff
@@ -8,12 +8,11 @@
...
 
-
 const getMediaButton = ($editor) => context => (
-    $.summernote.ui.button({
-        contents: $.summernote.ui.icon($.summernote.options.icons.picture),
-        tooltip: $.summernote.lang[$.summernote.options.lang].image.image,
-        click() {
+  $.summernote.ui.button({
+    contents: $.summernote.ui.icon($.summernote.options.icons.picture),
+    // tooltip: $.summernote.lang[$.summernote.options.lang].image.image, //Temporarily disabled
```
[the remark this answers is not in the document]
Comment (tmskrtsz) [reply shuup/admin/static_src/base/js/editor.js]: For some reason it `$.summernote.lang[$.summernote.options.lang].image.image` in that particular place returns as undefined. Console logging the return of that statement either before or after returns the string as it should. I haven't figured out a solution for that yet.

File: shuup/admin/static_src/base/assets/logo.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (chessbr) [reply shuup/admin/static_src/base/assets/logo.svg]: LOL, true, this **should** be

File: shuup/admin/static_src/picotable/picotable.js [hunk 3]
```diff
@@ -312,27 +353,47 @@ const Picotable = (function(m, storage) {
...
         });
-        return m("div.text-filter", input);
+        return m("div.text-filter", [label, input]);
     }
 
-    function buildColumnFilter(ctrl, col) {
+    // Check to see if it's any of the types of filters that
+    // we want to highlight by placing it at the top of the table
+    function isLiftFilter(col) {
+      return col.id === "name" || col.id === "customer" || col.id === "title" || col.id ==="code"
```
[the remark this answers is not in the document]
Comment (tmskrtsz) [reply shuup/admin/static_src/picotable/picotable.js]: It's a dirty solution for prototyping purposes. I'm going to refactor that.

**Pull request closed without merging**